docs/topic-guides/hp-tuning-det/hp-adaptive-asha.txt

```diff
@@ -51,16 +49,176 @@ Resource budget:
  Details
 *********
 
-Adaptive (ASHA) is an approximation to the resource allocation scheme
-used by :ref:`Adaptive <topic-guides_hp-tuning-det_adaptive-advanced>`.
-While Adaptive promotes hyperparameter configurations synchronously,
-resulting in underutilized nodes waiting on completion of validation
-steps for other configurations, ASHA uses asynchronous promotions to
-maximize compute efficiency of the searcher.
+Conceptually, the ``adaptive_asha`` searcher is a carefully tuned
+strategy for spawning multiple *ASHA* (asynchronous successive halving
+algorithm) searchers, themselves hyperparameter search algorithms. ASHA
+can be configured to make different tradeoffs between exploration and
+exploitation, i.e., how many trials are explored versus how long a
+single trial is trained for. Because the right tradeoff between
+exploration and exploitation is hard to know in advance, the
+``adaptive_asha`` algorithm tries several SHA searches with different
+tradeoffs.
+
+The configuration settings available to Determined experiments running
+in ``adaptive_asha`` mode mostly affect the ASHA subroutines directly.
+The ``mode`` configuration is the only one affecting the decisions of
+the ``adaptive_asha`` searcher, by changing the number and types of ASHA
+subroutines spawned.
+
+The first section here gives a description of ASHA. The second section
```

Comment:
Proposed reorg:
Switch current sections 2 and 3. 

Proposed edit:
The first section here gives a description of the synchronous version of ASHA called successive halving.  The second section discusses the motivation for the asynchronous promotions used by ASHA.  The third section describes why you would choose ``adaptive_asha`` over plain ``asynchronous_halving``.